extensions/lifecycle/tasks/LifecycleTask.js

```diff
@@ -95,11 +95,7 @@ class LifecycleTask extends BackbeatTask {
             timeProgressionFactor,
         });
 
-        this._lifecycleUtils = new LifecycleUtils(
-            this.supportedRules,
-            this._lifecycleDateTime
-        );
-        this._supportedRules = this.supportedRules;
+        this.setSupportedRules(this.supportedRules);
```

Comment:
We should test this in some unit test to fix the codecov warning? (same for other places)